deno.jsonc

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://deno.land/x/deno/cli/schemas/config-file.v1.json",
   "fmt": {
     "files": {
-      "include": ["src", "tests"],
+      "include": ["src", "tests", "docs", "README.md"],
```

Comment:
Do we also wanna format the generation script readme too?

Reply:
Yeah, but since it's contained in `src` it's already getting formatted. I did remove the exclusion of `**/*.md` files though